pkg/ansible/watches/watches.go

```diff
@@ -171,7 +172,7 @@ func (w *Watch) setValuesFromAlias(tmp alias) error {
 	w.Role = tmp.Role
 	w.Vars = tmp.Vars
 	w.MaxRunnerArtifacts = tmp.MaxRunnerArtifacts
-	w.MaxWorkers = getMaxWorkers(gvk, maxWorkersDefault)
+	w.MaxWorkers = getMaxReconciles(gvk, maxReconcilesDefault)
```

Comment:
Will modify the variable naming for `watches` in a follow up PR along with helm modifications.